Assets/Scripts/UI/SlipTimeUI.cs

```diff
@@ -1,33 +1,49 @@
-﻿using SlipTime;
+﻿using System;
+using SlipTime;
 using UnityEngine;
-using UnityEngine.Serialization;
 using UnityEngine.UI;
 
 namespace UI
 {
     public class SlipTimeUI : MonoBehaviour
     {
         public SlipTimeManager slipTimeManager;
-    
+
         public Text slipTimeUIText;
 
         private int slipTimeCharges;
 
         // Start is called before the first frame update
-        void Start()
+        private void Start()
         {
             slipTimeCharges = slipTimeManager.slipTimeCharges;
             slipTimeUIText.text = $"Slip Time Charges: {slipTimeCharges}";
         }
 
         // Update is called once per frame
-        void Update()
+        private void Update()
         {
-            if (slipTimeCharges != slipTimeManager.slipTimeCharges)
+            var slipTimeText = $"Slip Time Charges: {slipTimeManager.slipTimeCharges}";
+            
+            // Leave this code here for now. We may be able to re-use it later once we get a better UI for SlipTime
+            // if we don't have to constantly update the text box. If this comment is confusing, talk to NLow.
+            // if (slipTimeCharges != slipTimeManager.slipTimeCharges)
+            // {
+            //     slipTimeCharges = slipTimeManager.slipTimeCharges;
+            //     slipTimeUIText.text = $"Slip Time Charges: {slipTimeCharges}";
+            // }
```

Comment:
Basically, this `if` statement was here so that we didn't have to override/update the UI text box every update frame if we didn't have to. Since we're using the same text box to report duration/cooldown values right now, we pretty much have to update it every frame, and the logic is much simpler without this `if` statement. We can go back to the optimized solution if we can when we get the real GUI for SlipTime.

Reply:
Okay, sounds good.